pyramus/updates/39.001/160527_lodgingfee.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<update xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.ofw.fi/xml/2011/java-xmldb-updater/UpdaterSchema.xsd">
+
+  <sql>
+    alter table CourseStudent add lodgingFee double precision
```

Comment:
Use decimal numbers, using floating point for currency leads to rounding errors.